README.md

```diff
@@ -1,4 +1,4 @@
-# RTree [![Build Status](https://travis-ci.org/Workiva/r_tree.svg)](https://travis-ci.org/Workiva/r_tree)
```

Comment:
Even though this is a readme only change, I prefer every PR to have a description with an explanation of why these change are being done, however brief.